lib/generators/templates/rockauth_full_initializer.rb

```diff
@@ -23,6 +23,10 @@
   config.email_from = 'change-me-in-config-initializers-rockauth-rb@please-change-me.example'
   # config.forgot_password_always_successful = false
 
+  # config.filter_include = -> controller, is_collection do # Pass include from params to render for active model serializers. Default is only to pass nil
+  #   controller.params[:include]
+  # end
+
```

Comment:
Template generator. Example but disabled configuration similar to what you'll find in a freshly generated `routes.rb` or devise initializer.